lib/internal/Magento/Framework/Pricing/Render/Amount.php

```diff
@@ -217,14 +227,36 @@ protected function getAdjustments($adjustmentRenders)
      * @param float $amount
      * @param bool $includeContainer
      * @param int $precision
+     * @param null|string|bool|int|\Magento\Framework\App\ScopeInterface $scope
+     * @param \Magento\Framework\Model\AbstractModel|string|null $currency
      * @return float
      */
     public function formatCurrency(
         $amount,
         $includeContainer = true,
-        $precision = PriceCurrencyInterface::DEFAULT_PRECISION
+        $precision = null,
+        $scope = null,
+        $currency = null
     ) {
-        return $this->priceCurrency->format($amount, $includeContainer, $precision);
+        if ($precision === null) {
+            $precision = $this->getPricePrecision();
+        }
+
+        return $this->priceCurrency->format($amount, $includeContainer, $precision, $scope, $currency);
+    }
+
+    /**
+     * @return int
+     */
+    protected function getPricePrecision()
```

Comment:
Please change visibility to private for backward compatibility purpose

Reply:
@sivaschenko done